docs/guide/getting-started.md

```diff
@@ -306,7 +306,7 @@ asdf global nodejs latest
 nodejs 16.5.0
 ```
 
-Some OSs already have tools installed that are managed by the system and not `asdf`, `python` is a common example. You need to tell `asdf` to pass the management back to the system. The [Versions reference section](/manage/versions.md) will guide you.
+Some OSs already have tools installed that are managed by the system and not `asdf`, `python` is a common example. You need to tell `asdf` to pass the management back to the system. The [Versions reference section](/docs/manage/versions.md) will guide you.
```

Comment:
It's not clear to my why this `/docs/` prefix is needed. I believe the canonical URL is `https://asdf-vm.com/manage/versions.html` so I would think that would translate to `/manage/versions.md` here.

Reply:
When I navigate here: 
https://github.com/asdf-vm/asdf/tree/master/docs/manage/versions.md
success

When I navigate here:
https://github.com/asdf-vm/asdf/tree/master/manage/versions.md
404 not found

Reply:
I may need a quick

How to on what tool is used if the docs are generated.
as these are hardcoded changes to the Readme files on github.

@Stratus3D , thanks for taking a look at this!

Reply:
Please read this page - http://asdf-vm.com/contribute/documentation.html#initial-setup

The files in the `docs/` directory are used by vuepress to generate the documentation website. The links will not work when viewing the source code on Github.

Reply:
So the pr should just be closed?

interacting with the docs on github is a helpful user experience since you can search across the entire repo

I will look at initial setup and see if there is anything I can do to fix github without breaking vuepress.

it might make sense for me to close the mr and make a new mr with just the https.

Reply:
The only change I see here that I think we need is the `http` => `https` in the readme. All other changes can be discarded.